trino/dbapi.py

```diff
@@ -454,7 +488,8 @@ def genall(self):
         return self._query.result
 
     def fetchall(self) -> List[List[Any]]:
-        return list(self.genall())
+        res = list(self.genall())
+        return res
```

Comment:
Why this change?

Reply:
Rollback. I changed it to set break-point while debugging.